Dockerfiles/agent/entrypoint.d/process-agent

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+exec /opt/datadog-agent/embedded/bin/process-agent -config=/etc/datadog-agent/datadog.yaml
```

Comment:
I’ve just added it in https://github.com/DataDog/datadog-agent/pull/8490/commits/97fb88f11b5825cae808cf75daec3a3c50714e06.
We didn’t have it in the Helm chart: [charts/datadog/templates/_container-process-agent.yaml](https://github.com/DataDog/helm-charts/pull/289/files#diff-abf9fdd0c458bac795b56656500aa020684d56b7172d3e2e9b0bdb970d31304cL6)
I think both are correct because, according to the `process-agent` help, the default value is already the value that we specify explicitely:
```
root@datadog-agent-linux-95crf:/# process-agent --help
Usage of process-agent:
[…]
  -config string
    	Path to datadog.yaml config (default "/etc/datadog-agent/datadog.yaml")
[…]
  -sysprobe-config string
    	Path to system-probe.yaml config (default "/etc/datadog-agent/system-probe.yaml")
```